lib/utils/parse/parse.go

```diff
@@ -19,10 +19,10 @@ limitations under the License.
 package parse
```

Comment:
Can this TODO be removed now?

Reply:
Not yet. `regexp.match` cannot yet contain expressions, only strings.
I can try to fix it as part of this PR or in a follow up.

Reply:
Removed this TODO in ae1ce37dda027045b873d57d7d45b7c89e109085 as it's now possible to write e.g. `{{regexp.match(email.local(vitor@goteleport.com))}}`.

What's not yet implemented is variable interpolation inside match expressions, but limitation is already documented in `buildRegexpMatchFnExpr`'s TODO (`ast.go`).

Reply:
Added the TODO back as, while it's possible to write `{{regexp.match(email.local("vitor@goteleport.com"))}}` without variable interpolation, this is not implemented yet.